Build/Customizations/SqlServer/WindowsSqlServerDockerImage.cs

```diff
@@ -0,0 +1,57 @@
+// SPDX-FileCopyrightText: (c) RUBICON IT GmbH, www.rubicon.eu
+// SPDX-License-Identifier: LGPL-2.1-or-later
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using DotNet.Testcontainers.Builders;
+using DotNet.Testcontainers.Containers;
+using DotNet.Testcontainers.Networks;
+using JetBrains.Annotations;
+using Serilog;
+
+namespace Customizations.SqlServer;
+
+public class WindowsSqlServerDockerImage : ISqlDockerImage
+{
+  private readonly IContainer _container;
+  private readonly string _connectionString;
+
+  public WindowsSqlServerDockerImage (string imageName, string name, [CanBeNull] INetwork network)
+  {
+    const string password = "P@ssw0rd";
```

Comment:
Don't we get the password from teamcity?

Before this change, the password wasn't hardcoded in the build script, if I remember correctly. And while that shouldn't be an issue, it still feels strange thing to do.

Should we use a Guid as a password instead (new'd each time)?